src/test/java/org/hydev/mcpm/utils/HashUtilsTest.java

```diff
@@ -0,0 +1,37 @@
+package org.hydev.mcpm.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Objects;
+
+class HashUtilsTest
+{
+    HashUtils hashUtils = new HashUtils();
+
+    @Test
+    void hashString()
+    {
+        assert hashUtils.hash("hello world")
```

Comment:
Might just be best to go through all `assert`s by hand 😳

Reply:
yea right